jarvis/jasper_v0.8.2_chkpt_to_onnx.py

```diff
@@ -0,0 +1,80 @@
+import nemo
```

Comment:
@ryanleary let's delete this file

Reply:
Yes I was going to add a flag in the other file